tox.ini

```diff
@@ -1,49 +1,30 @@
 # The test environment and commands
 [tox]
-envlist = deps, check, test
+envlist = check, test
 skipsdist = True
 
 [testenv:check]
 description = Runs all formatting tools then static analysis (quick)
-deps =
-    --no-deps
-    -r requirements-dev.txt
+deps = flake8
 commands =
     #shed  # combines autoflake, black, isort, and pyupgrade
     flake8
 
-[testenv:test]
+[testenv]
 description = Run the tests
 deps =
-    --no-deps
-    -r requirements.txt
+     pytest
+     pytest-cov
+     pytest-mock
+     pytest-xdist
 commands =
     pip install -e .
     python -m pytest --tb=native --cov=terminusdb_client terminusdb_client/tests/{posargs} --cov-report xml:cov.xml
 passenv = TERMINUSX_TOKEN
 
 
-[testenv:deps]
-description = Update pinned requirements
-deps =
-    pip-tools
-setenv =
-    CUSTOM_COMPILE_COMMAND = tox -e deps
-commands =
-    pip-compile --quiet --upgrade --rebuild --output-file=requirements.txt requirements.in
-    pip-compile --quiet --upgrade --rebuild --output-file=requirements-dev.txt requirements-dev.in
-
-
 # Settings for other tools
 [flake8]
 # S501 is turned off for now, check if the verify=false for request is ok
 ignore = E501,W503,S101,S310,A003,E203,S607,S603,S404,W391,N814,N804,S106,F722,S105,E266,S311,N806
 exclude = .*/,__pycache__,docs/,build/
-
```

Comment:
isort is now configurable through the pyproject.toml file, which imo is better.

I'd have liked to move everything there. Unfortunately, flake8 cannot yet do it. Therefore there's still a flake8 section above.